### PR TITLE
feat: Switch to Ethical Ads

### PR DESCRIPTION
**Pull request by nzakas**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request?

Switches our ad to Ethical Ads from Carbon Ads.

#### What changes did you make? (Give an overview)

Migrated the site code to use Ethical Ads. Due to #520, this change is only visible on the homepage but will be visible on all pages once that is fixed.

We'll also need to duplicate this on the docs site.

### CODE REVIEW

**Comment by harish-sethuraman** [comment]
Have duplicated the same in eslint repo https://github.com/eslint/eslint/pull/18090

File: src/_data/sites/en.yml [hunk 1]
```diff
@@ -40,9 +40,8 @@ google_analytics:
 # Ads
 #------------------------------------------------------------------------------
 
-carbon_ads:
-  serve: CESDV2QM
-  placement: eslintorg
```
Comment (harish-sethuraman) [inline src/_data/sites/en.yml:45]: should we consider doing the same here? keep serve and placement?

File: src/assets/scss/components/ads.scss [hunk 2]
```diff
@@ -120,3 +125,39 @@
...
+[data-ea-type].loaded .ea-content {
+    background-color: var(--body-background-color) !important;
+    border: 1px solid var(--border-color) !important;
+}
+
+// [data-ea-publisher].loaded .ea-content a:hover,
+// [data-ea-type].loaded .ea-content a:hover {
+//     text-decoration: none;
+//     color: var(--link-color);
+// }
```
Comment (harish-sethuraman) [inline src/assets/scss/components/ads.scss:143]:
```suggestion
```

**Comment by mdjermanovic** [comment]
This switches ads on eslint.org only. Shall we do the same for translation sites (e.g., es.eslint.org) as well?

**Comment by nzakas** [comment]
@mdjermanovic yes, just needed to verify that it was okay to put the ads there, and it is.

**Review by harish-sethuraman — APPROVED** [review]
LGTM

**Pull request merged**